pkg/postprocess/skipAccountTokenPassword.go

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 American Express
```

Comment:
Change to `2021`?

Reply:
also added to files where it was missing